R/pull_GA_data.R

```diff
@@ -158,6 +158,10 @@ write_df_to_parquet(regionality_metric,
                     sink = 'out/regionality/regionality.parquet')
 
 state_week_vs_year <- compute_week_vs_year(state_traffic_all)
+state_week_vs_year <- state_week_vs_year %>% 
```

Comment:
can you just pipe line 160 into the rename commands below?  You can do multiple renames within one function call as well.

Reply:
yes, thanks for the suggestions.